app-web/registry/data-and-apis.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "Data and APIs",
+  "description": "Privince provided Data and API Services",
+  "sourceProperties": {
+    "sources": [
+      {
+        "sourceType": "github",
+        "sourceProperties": {
+          "url": "https://github.com/bcgov/api-guidelines",
+          "owner": "bcgov",
+          "repo": "api-guidelines",
+          "files": [
+            "government-of-british-columbia-api-guidelines.md#api-performance-management",
```

Comment:
this doesn't work as you think it does. `files` should just have the one md. file.

Reply:
ah, ok.